perun-core/src/test/java/cz/metacentrum/perun/core/entry/ConsentsManagerEntryIntegrationTest.java

```diff
@@ -967,21 +822,53 @@ public void createConsentNotIncludeAttrFromExpiredServices() throws Exception {
 	private ConsentHub setUpAnotherConsentHub() throws Exception {
```

Comment:
Not quite sure what these changes are for. Couldn't we delete this method, since it was only used in the tests that you deleted?

Reply:
Thank you for your suggestions! I edited the javadoc in several layers, openAPI including.
As for the test part, I wanted to leave it there for future needs (if needed), however, it makes more sense to remove it now, since it is not used at all.